x11-misc/imake/files/imake-1.0.8-no-get-gcc.patch

```diff
@@ -0,0 +1,37 @@
+If /usr/bin/cc exists then get_gcc() is always true on Linux (and many
+others), but will fail in the event it's missing (-native-symlinks).
+
+get_gcc_version does not execute gcc and merely sets defines, so
+call it without using get_gcc(). Validity of using NULL should be
+verified if there's ever a new version.
+--- a/imake.c
```

Comment:
Bug reference for this for posterity?

Reply:
There's no bug considering had to fix the other bugs to hit this issue.